netcore/Makefile

```diff
@@ -164,9 +166,8 @@ run-tests-corefx-%: prepare update-tests-corefx
 
 # build a test assembly in COREFX_ROOT and copy it to corefx/tests/extracted
 # e.g. `make build-test-corefx-System.Runtime COREFX_ROOT=/prj/corefx-master`
-build-test-corefx-%: check-env
-	cd $(COREFX_ROOT)/src/$*/tests && $(DOTNET) build -o "$(CURDIR)/corefx/tests/extracted/tmp"
-	cp $(CURDIR)/corefx/tests/extracted/tmp/$*.Tests.{dll,pdb} $(CURDIR)/corefx/tests/extracted/$*.Tests/
-	rm -rf $(CURDIR)/corefx/tests/extracted/tmp
+build-test-corefx-%:
+	cd $(COREFX_ROOT)/src/$*/tests && $(DOTNET) msbuild /p:OutputPath=tmp
```

Comment:
Will it trigger the whole CoreFX build in case of the first pull? If no - maybe does it make sense to add short comment that CoreFX full build is required?